Auto update failing for proxmark3

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the proxmark3 package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating proxmark3 [Current version: 1:4.16717]
INFO: package being updated to 4.17140.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2320k    0 2320k    0     0  2738k      0 --:--:-- --:--:-- --:--:-- 2738k
100 5552k    0 5552k    0     0  2984k      0 --:--:--  0:00:01 --:--:-- 3190k
100 9504k    0 9504k    0     0  3340k      0 --:--:--  0:00:02 --:--:-- 3593k
100 24.2M    0 24.2M    0     0  6439k      0 --:--:--  0:00:03 --:--:-- 7485k
100 25.7M    0 25.7M    0     0  6333k      0 --:--:--  0:00:04 --:--:-- 7251k

You are about to commit these changes:

--------------------
diff --git a/packages/proxmark3/build.sh b/packages/proxmark3/build.sh
index aea133eda..ad2d885d4 100644
--- a/packages/proxmark3/build.sh
+++ b/packages/proxmark3/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
 TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@esque.ca>"
-TERMUX_PKG_VERSION="1:4.16717"
+TERMUX_PKG_VERSION="1:4.17140"
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
-TERMUX_PKG_SHA256=675a785e859d600b55de67ecd17a85ff26741d8e1b74c51ed0fa585850b44f3f
+TERMUX_PKG_SHA256=b38872d36f49fba4be76ee9c2691ec30583c1faad9616c3a1bd14f167f43bada
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
 TERMUX_PKG_BUILD_IN_SRC="true"
--------------------

bump(main/proxmark3): 4.17140


Not committing to Git!

INFO: Trying to build package.
INFO: Package proxmark3 exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building proxmark3 for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  32879      0 --:--:-- --:--:-- --:--:-- 32832
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2097      0 --:--:-- --:--:-- --:--:--  2098
gpg: Signature made Sat 09 Sep 2023 01:58:10 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  33205      0 --:--:-- --:--:-- --:--:-- 33131
100 13319  100 13319    0     0  33050      0 --:--:-- --:--:-- --:--:-- 33049
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2007      0 --:--:-- --:--:-- --:--:--  2012
gpg: Signature made Thu 07 Sep 2023 12:20:57 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  33077      0 --:--:-- --:--:-- --:--:-- 33045
100 13119  100 13119    0     0  33069      0 --:--:-- --:--:-- --:--:-- 33045
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2096      0 --:--:-- --:--:-- --:--:--  2098
gpg: Signature made Fri 01 Sep 2023 06:40:02 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@25c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20230527 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20230527 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20230527 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20230527 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/RfidResearchGroup/proxmark3/archive/v4.17140.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 40957    0 40957    0     0   124k      0 --:--:-- --:--:-- --:--:--  124k
100 3232k    0 3232k    0     0  2415k      0 --:--:--  0:00:01 --:--:-- 3141k
100 5920k    0 5920k    0     0  2544k      0 --:--:--  0:00:02 --:--:-- 2934k
100 9120k    0 9120k    0     0  2743k      0 --:--:--  0:00:03 --:--:-- 3024k
100 21.8M    0 21.8M    0     0  5164k      0 --:--:--  0:00:04 --:--:-- 5569k
100 25.7M    0 25.7M    0     0  5246k      0 --:--:--  0:00:05 --:--:-- 5596k
Applying patch: use-softfp.patch

===================================================================
Version info:      Iceman/master/release (git)
Platform name:     Proxmark3 generic target
PLATFORM:          PM3GENERIC
PLATFORM_FPGA:     xc2s30
PLATFORM_SIZE:     512
Platform extras:   No extra selected
Included options:  LF HITAG EM4x50 EM4x70 ZX8211 GENERAL_HF ISO15693 LEGICRF ISO14443b ISO14443a ICLASS FELICA NFCBARCODE HFSNIFF HFPLOT
Standalone mode:   LF_SAMYRUN
===================================================================
[!] Platform definitions changed, cleaning bootrom/armsrc/recovery first...
===================================================================
Version info:      Iceman/master/release (git)
Client platform:   Linux
GUI support:       skipped
native BT support: Bluez not found, disabled
Jansson library:   system library not found, using local library
Lua library:       system library not found, using local library
Python3 library:   Python3 not found, disabled
Readline library:  enabled
Whereami library:  system library not found, using local library
Lua SWIG:          wrapper found
compiler version:  Android (9352603, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)
===================================================================
PLATFORM Linux
[*] MAKE client/all
===================================================================
Version info:      Iceman/master/release (git)
Client platform:   Linux
GUI support:       skipped
native BT support: Bluez not found, disabled
Jansson library:   system library not found, using local library
Lua library:       system library not found, using local library
Python3 library:   Python3 not found, disabled
Readline library:  enabled
Whereami library:  system library not found, using local library
Lua SWIG:          wrapper found
compiler version:  Android (9352603, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)
===================================================================
[-] CC src/mifare/aiddesfire.c
[-] CC src/aidsearch.c
[-] CC src/atrs.c
[-] CC src/cmdanalyse.c
[-] CC src/cmdcrc.c
[-] CC src/cmddata.c
[-] CC src/cmdflashmem.c
[-] CC src/cmdflashmemspiffs.c
[-] CC src/cmdhf.c
[-] CC src/cmdhf14a.c
[-] CC src/cmdhf14b.c
[-] CC src/cmdhf15.c
[-] CC src/cmdhfcryptorf.c
[-] CC src/cmdhfepa.c
[-] CC src/cmdhfemrtd.c
[-] CC src/cmdhffelica.c
[-] CC src/cmdhffido.c
[-] CC src/cmdhffudan.c
[-] CC src/cmdhfgallagher.c
[-] CC src/cmdhfksx6924.c
[-] CC src/cmdhfcipurse.c
[-] CC src/cmdhficlass.c
[-] CC src/cmdhflegic.c
[-] CC src/cmdhfjooki.c
[-] CC src/cmdhflist.c
[-] CC src/cmdhflto.c
[-] CC src/cmdhfmf.c
[-] CC src/cmdhfmfdes.c
[-] CC src/cmdhfmfhard.c
[-] CC src/cmdhfmfu.c
src/cmdhfmfhard.c:33:10: fatal error: 'lz4frame.h' file not found
#include <lz4frame.h>
         ^~~~~~~~~~~~
1 error generated.
make[1]: *** [Makefile:941: obj/cmdhfmfhard.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:177: client/all] Error 2
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-09-09 12:18:18 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/6130754683">6130754683</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/proxmark3/build.sh
+++ b/packages/proxmark3/build.sh
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
 TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@esque.ca>"
-TERMUX_PKG_VERSION="1:4.16717"
+TERMUX_PKG_VERSION="1:4.17140"
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
-TERMUX_PKG_SHA256=675a785e859d600b55de67ecd17a85ff26741d8e1b74c51ed0fa585850b44f3f
+TERMUX_PKG_SHA256=b38872d36f49fba4be76ee9c2691ec30583c1faad9616c3a1bd14f167f43bada
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
+TERMUX_PKG_DEPENDS="libbz2, libc++, readline, liblz4"
 TERMUX_PKG_BUILD_IN_SRC="true"
 TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
 

